### PR TITLE
Upgrade to PHP 5.6

### PR DESCRIPTION
**Pull request by lo5an**
* Update d7 role to use Software Collections provided PHP 5.6
* Switch from mod_php to  PHP-FPM (simplifies upgrade). 
* Switch from APC to Zend OpCache (See the [APC death notice](http://php.net/manual/en/intro.apc.php).)

Requires the latest OULibraries.sclphp

Motivation and Context
----------------------
* php 5.4 is old. 
* software collections are cool
* PHP-FPM doesn't require also upgrading the CentOS Apache
* Apache + PHP-FPM gives us some great options for tuning performance. 

Testing/Status
-------------------------
~Still in dev.~
Testing in progress.
 *Not ready for merge*.

### CODE REVIEW

**Comment by lo5an** [comment]
This is a fairly open config and matches what we were doing with mod_php. One thing that we might consider as we move forward is locking down PHP execution to the 10 or so files that can bootstrap for Drupal 7.

File: templates/10-opcache.ini.j2 [hunk 1]
```diff
@@ -0,0 +1,95 @@
+; Enable Zend OPcache extension module
+zend_extension=opcache.so
+
+; Determines if Zend OPCache is enabled
+opcache.enable={{ d7_opcache_enable }}
+
+; Determines if Zend OPCache is enabled for the CLI version of PHP
+;opcache.enable_cli=0
```
Comment (jsnshrmn) [inline templates/10-opcache.ini.j2]: We should see what we're doing for apc on php cli caching, I immediately thought about caching stuff for drush bootstrap. Can't remember if we are doing that at oulib.

**Review by jsnshrmn — APPROVED** [review]
Looks good upon interpreting with /bin/eyeball

**Comment by lo5an** [comment]
We're moving forward with mod_php.

**Pull request closed without merging**